diff --git a/node_modules/video.js/dist/video.es.js b/node_modules/video.js/dist/video.es.js
index 57afd7a..a9a3e4c 100644
--- a/node_modules/video.js/dist/video.es.js
+++ b/node_modules/video.js/dist/video.es.js
@@ -27779,6 +27779,8 @@ class QualityLevel {
     level.bitrate = representation.bandwidth;
     level.frameRate = representation.frameRate;
     level.enabled_ = representation.enabled;
+    level.price = representation.playlist.custom?.price;
+    level.resolvedUri = representation.playlist.resolvedUri;
     Object.defineProperty(level, 'enabled', {
       /**
        * Get whether the QualityLevel is enabled.
